synapse/models/dns.py

```diff
@@ -204,7 +204,7 @@ def getBaseModels():
 
                 ('inet:dns:look', {'ptype': 'inet:dns:look', 'doc': 'Instance knowledge of a DNS record lookup.'}, [
 
-                    ('time', {'ptype': 'time', 'req': 1, 'ro': 1,
+                    ('time', {'ptype': 'time', 'ro': 1,
```

Comment:
This is modified to make it consistent with other elements that are GUID primary props and have time.  Time shouldn't be mandated when using stable guid generation using other fields...